src/lib/config.js

```diff
@@ -23,42 +23,42 @@ export const registrationStart =
  *   - Participation page information
  */
 export const registrationEnd =
-  process.env.REGISTRATION_END || '2023-11-01T12:00:00Z';
+  process.env.REGISTRATION_END || '2024-11-01T12:00:00Z';
 
 /**
  * Used for:
  *   - Toggling no PR/MR messaging on profile
  *   - Participation page information
  */
 export const trackingStart =
-  process.env.TRACKING_START || '2023-09-30T10:00:00Z';
+  process.env.TRACKING_START || '2024-10-01T10:00:00Z';
```

Comment:
```suggestion
  process.env.TRACKING_START || '2024-09-30T10:00:00Z';
```
Tracking starts as soon as we hit October 1st in the first timezone.